src/components/ChromeWindow.tsx

```
@@ -0,0 +1,451 @@
+    startH: number;
+    startPosX: number;
+    startPosY: number;
+  }>({ isResizing: false, edge: "", startX: 0, startY: 0, startW: 0, startH: 0, startPosX: 0, startPosY: 0 });
+  const prevSizeRef = useRef({ width: defaultWidth, height: defaultHeight, x: 0, y: 0 });
+  const currentSizeRef = useRef({ width: defaultWidth, height: defaultHeight });
+  const currentPosRef = useRef(position);
+  const prevMinimizedRef = useRef(minimized);
+  const MIN_WIDTH = 300;
+  const MIN_HEIGHT = 200;
+
+  currentSizeRef.current = size;
```

ESLint

[error] Error: Cannot access refs during render

React refs are values that are not needed for rendering. Refs should only be accessed outside of render, such as in event handlers or effects. Accessing a ref value (the `current` property) during render can cause your component not to update as expected (https://react.dev/reference/react/useRef).

/home/jailuser/git/src/components/ChromeWindow.tsx:125:3
  123 |   const MIN_HEIGHT = 200;
  124 |
> 125 |   currentSizeRef.current = size;
      |   ^^^^^^^^^^^^^^^^^^^^^^ Cannot update ref during render
  126 |   currentPosRef.current = position;
  127 |
  128 |   // Opening animation - runs once on mount

(react-hooks/refs)